JPF/utils.py

```diff
@@ -239,7 +242,7 @@ def send_df_to_sql(df, log, description, table_name, db_action, sql_chunk_size,
     df.to_sql(name=table_name, con=conn, if_exists=action,\
               index=False, chunksize=sql_chunk_size, dtype=outputdict)
 
-    if db_action == 'replace':
+    if db_action == 'replace' and set_default_primary_key:
         conn.execute("ALTER TABLE %s ADD id INT PRIMARY KEY AUTO_INCREMENT;" % table_name)
```

Comment:
Yesh. It should be that easy.

After a quick google, it seems there is a newer, clearer syntax for [adding indexes in MySQL](http://dev.mysql.com/doc/refman/5.7/en/create-index.html). We may want to cut over to that, since it's cleared.